function handleError(message) {
    alert(message);
}